motr/obj.c

```diff
@@ -555,6 +555,21 @@ static int obj_namei_op_init(struct m0_entity *entity,
 	return M0_ERR(rc);
 }
 
+/**
+ * Finds and sets optimal layout id according to the object's size.
+ */
+static void obj_optimal_lid_set(struct m0_obj *obj,
+				struct m0_layout_domain *ldom)
+{
+	uint64_t                *lid;
+
+	lid = &obj->ob_attr.oa_layout_id;
+
+	if (obj->ob_attr.oa_layout_id <= 0)
```

Comment:
The checking for < is needed, it can be obj->ob_attr.oa_layout_id == 0 ?

Reply:
Done.